docs/UserGuide.md

```diff
@@ -104,6 +104,14 @@ If you have a new client interested in your fitness training services, FitEgo ca
 
 All [client-related commands](#32-client-related-keywords) use prefix `c`. 
 
+<div markdown="block" class="alert alert-info"> 
+
+:information_source: **Note:**
+FitEgo will not allow you to have clients with the same email. This is because we believe that email is the most
+suitable identifier to differentiate clients.
```

Comment:
is this too technical/revealing the implementation?

Reply:
Perhaps it is better to just say "Each client should have a unique email"